pkg/cloudclient/aws/private.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	cloudingressv1alpha1 "github.com/openshift/cloud-ingress-operator/pkg/apis/cloudingress/v1alpha1"
 	machineapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	"github.com/sirupsen/logrus"
```

Comment:
i think this was imported by mistake?

Reply:
This is used in the copy/pasted function:
https://github.com/openshift/cloud-ingress-operator/pull/143/files#diff-0dfb65e2caeb4c025cfe6a19c3ed8e9add53107d57033e1be141011478911681R421